_documentation/05-User-Manual.md

```diff
@@ -1,5 +1,32 @@
 ---
 title: User Manual
+toc: true
+toc_sticky: true
+h_range: [1,3]
 ---
 
-This document will be an user manual, containing information about setting up cups and printer application drivers.
\ No newline at end of file
+**This document will be an user manual, containing information about installing printer/scanner application drivers, installing CUPS-Snap, configuring and using Web Interface options and finally use the printer or scanner to print and scan respectively.**
```

Comment:
This document will be **a** user manual, containing information about installing **drivers provided as Printer/Scanner Applications**, installing the CUPS Snap, configuring and using Web Interface options and finally use the printer or scanner to print and scan respectively.